include/dsAudio.h

```diff
@@ -2063,6 +2063,30 @@ dsError_t  dsGetSecondaryLanguage(intptr_t handle, char* sLang);
  */
 dsError_t dsGetHDMIARCPortId(int *portId);
 
+/**
+* @brief Sets the Mixer Volume level for the MS12 input
```

Comment:
Sets the Mixer Volume level for the MS12 input -> Why is this brief mentioning ms12, but the function defination does not?

Is the setAudoMixerLevels also for non ms12 streams?

Reply:
dsSetMS12AudoMixerLevels

Reply:
No this should be a generic mixer API.  In amlogic platform, it's going to MS12

Reply:
Updated as generic mixer and removed MS12 from header and enum briefs